TYPO3.Media/Documentation/AsynchronousThumbnails.rst

```diff
@@ -0,0 +1,41 @@
+=================================
+Asynchronous Thumbnail Generation
+=================================
+
+Introduction
+------------
+
+To optimize response times, generation of thumbnails can be done asynchronously. Usage of asynchronous thumbnail
+generation is determined in the image view helpers usage with the ``async`` flag. When the flag is used, a link to the
+thumbnail controller returned instead of the generating the thumbnail if the thumbnails hasn't already been generated.
```

Comment:
instead of generating the thumbnail if the thumbnail hasn't already been generated